geode-core/src/test/java/org/apache/geode/distributed/ServerLauncherRemoteIntegrationTestCase.java

```diff
@@ -0,0 +1,270 @@
+  }
+
+  protected ServerLauncher awaitStart(final ServerCommand command) {
+    try {
+      executeCommandWithReaders(command);
+      ServerLauncher launcher = awaitStart(getWorkingDirectory());
+      assertThat(process.isAlive()).isTrue();
+      return launcher;
+    } catch (IOException e) {
+      throw new UncheckedIOException(e);
+    }
+  }
+
+  protected ServerLauncher awaitStart(final File workingDirectory) {
+    try {
+      this.launcher = new ServerLauncher.Builder()
+          .setWorkingDirectory(workingDirectory.getCanonicalPath()).build();
+      ServerLauncher launcher = awaitStart(this.launcher);
+      assertThat(process.isAlive()).isTrue();
+      return launcher;
+    } catch (IOException e) {
+      throw new UncheckedIOException(e);
+    }
+  }
+
+  @Override
+  protected ServerLauncher awaitStart(final ServerLauncher launcher) {
+    await().until(() -> assertThat(launcher.status().getStatus()).isEqualTo(Status.ONLINE));
+    assertThat(process.isAlive()).isTrue();
+    return launcher;
+  }
+
+  protected ServerLauncher awaitStart() {
+    return awaitStart(this.launcher);
+  }
+
+  protected void awaitStop() {
+    await().until(() -> assertThat(this.process.isAlive()).isFalse());
+  }
+
+  protected void assertStopOf(final Process process) {
+    await().until(() -> assertThat(process.isAlive()).isFalse());
+  }
+
+  protected Process getServerProcess() {
+    return this.process;
+  }
+
+  protected void startServerShouldFail(final ServerCommand command, final InputListener outListener,
+      final InputListener errListener) throws IOException, InterruptedException {
+    executeCommandWithReaders(command.create(), outListener, errListener);
+    process.waitFor(2, MINUTES);
+    assertThat(process.isAlive()).isFalse();
+    assertThat(process.exitValue()).isEqualTo(1);
+  }
+
+  protected void startServerShouldFail(final ServerCommand command)
+      throws IOException, InterruptedException {
+    startServerShouldFail(command, createBindExceptionListener("sysout", threwBindException),
+        createBindExceptionListener("syserr", threwBindException));
+  }
+
+  protected void startServerShouldFail() throws IOException, InterruptedException {
+    startServerShouldFail(serverCommand);
+  }
+
+  private InputListener createBindExceptionListener(final String name,
+      final AtomicBoolean threwBindException) {
+    return createExpectedListener(name, getUniqueName() + "#" + name, BindException.class.getName(),
+        threwBindException);
+  }
+
+  @Override
+  protected ServerLauncher startServer() {
+    return startServer(serverCommand);
+  }
+
+  protected void assertThatProcessIsNotAlive() {
+    assertThatProcessIsNotAlive(process);
+  }
+
+  protected void assertThatProcessIsNotAlive(final Process process) {
+    assertThat(process.isAlive()).isFalse();
+  }
+
+  protected void assertProcessIsNotAlive() {
+    assertThat(process.isAlive()).isFalse();
+  }
+
+  protected void assertProcessIsAlive() {
```

Comment:
Delete unused method. Etc. I'm not gonna mark the rest of these... just delete them if they're unused.